helm/observability-bundle/values.yaml

```diff
@@ -65,6 +65,26 @@ apps:
         name: "{{ $.Values.clusterID }}-logging-config"
         namespace: "{{ $.Release.Namespace }}"
 
+  alloyMetrics:
+    appName: alloy-metrics
+    chartName: alloy
+    catalog: giantswarm-test
+    enabled: false
+    namespace: kube-system
+    # used by renovate
+    # repo: giantswarm/alloy-app
+    version: 0.3.1-0ea7e6c9fea5fa4ce5aede8f9e20ea21448fe0fb
```

Comment:
I assume this version will have to be changed :D

Reply:
Updated to v0.4.0